backend/db-services.js

```diff
@@ -180,6 +180,25 @@ async function deleteItem(invId, itemId) {
   }
 }
 
+async function updateItem(invId, itemId, colName, value) {
+  if (!invId || !itemId) throw new Error("Invalid id");
+  const inventory = await Inventory.findById(invId);
+  if (inventory === null) throw new Error("Inventory not found");
+  let col = inventory["columnNames"].indexOf(colName);
+  if (col === -1) {
+    console.log("Invalid column");
+  } else {
+    // This goes through all of the items in the Inventory Table to find the matching item Id
+    const result = await Inventory.findById(invId);
```

Comment:
You already declared `inventory` with the same value so you should use that instead of the `result` variable